### ISSUE TITLE
Admin users permissions for separate documents

### ISSUE THREAD

**Issue by sakaljurgis** [litecart/litecart#243]
**_What were you trying to do?_**
Edit user permissions for separate document in admin users.app e.g. allow only **search** in translations, but not **scan** or **csv**.

**_What was expected to happen?_**
After saving - all is good, permissions are set correctly. But if I want to edit that user again, I expect that in translations only **search** is selected.

**_What was the actual result - what exactly did happen?_**
But unfortunately in the edit user all options are selected for the app - I need to de-select again **scan** and **csv** leaving only  **search** in translations.

**_Which LiteCart version are you running?_**
2.1.5. I have compared edit_user.inc.php for v 2.2.3.6 and it seems that permission logic is the same. Unfortunately permissions are disabled in demo, so unable to replicate.

**Comment by timint** [comment]
I can't reproduce I saved these settings. Then went back to the screen and it seems ok. What browser are you using? Just to make sure this is not caused by browser extensions. Open an incognito window without extensions enabled and see if you have the same issue.

![image](https://user-images.githubusercontent.com/359192/93268804-237d3600-f7ae-11ea-976e-fb69e6aa5d96.png)

**Comment by sakaljurgis** [comment]
I've found what was causing this. Apparently my comparison of `edit_user.inc.php` between `v2.1.5` and `2.2.3.6` was incomplete. 

The returned html provided all inputs correctly checked, but it was later changed in the browser by the javascript part.
Somehow a change was triggered.

Anyway I've copied the javascript part from `v2.2.3.6` and now everything works fine.

I am closing this as it is already solved in current version.
